automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -0,0 +1,158 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.test.util.AuthDomainMatcher
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.util.Tags
+import org.broadinstitute.dsde.workbench.service.{AclEntry, Orchestration, Rawls, WorkspaceAccessLevel}
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+
+class AuthDomainGroupApiSpec extends FreeSpec with Matchers with WorkspaceFixtures with BillingFixtures with GroupFixtures {
+
+  /*
+  * Unless otherwise declared, this auth token will be used for API calls.
+  * We are using a curator to prevent collisions with users in tests (who are Students and AuthDomainUsers), not
+  *  because we specifically need a curator.
+  */
+
+  val defaultUser: Credentials = UserPool.chooseCurator
+  val authTokenDefault: AuthToken = defaultUser.makeAuthToken()
```

Comment:
defaultUser and authTokenDefault should be removed, they're not being used anywhere

Reply:
`authTokenDefault` is used. line 81 for example.

Reply:
ah! You're right, I think I was thinking of a different PR. Sorry about that :)